### PR TITLE
Update ava to latest version

### PR DESCRIPTION
**Pull request by DavidTanner**
know a guy

### CODE REVIEW

File: src/docker.js [hunk 1]
```diff
@@ -113,8 +114,7 @@ exports.getHostAddress = async () => {
     const { stdout: ifconfig } = await executeContainerCommand({ container, command: ['ifconfig', defaultInterface] });
     return getInterfaceAddress(ifconfig.toString('utf8'));
   } finally {
-    // Don't wait for the container to stop (this can take a while).
-    container.stop();
```
Comment (mdlavin) [inline src/docker.js:117]: Is this not needed anymore?

File: src/localstack.ts [hunk 2]
```diff
@@ -411,9 +420,9 @@ export async function getConnection <Service extends keyof LocalStackServices>({
     mappedServices,
     getOutput: () => output.toString(),
     clearOutput: () => output.reset(),
-    cleanup: () => {
+    cleanup: async () => {
```
Comment (mdlavin) [inline src/localstack.ts:423]: Curious, why this change? In most of our projects the linter would complain that this is creating unneeded Promises

File: test/docker/executeContainerCommand.test.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ const { executeContainerCommand } = require('../../src/docker');
 
 test('response includes stdout, stderr, and inspectOutput with ExitCode 0', async (t) => {
   const container = await createDefaultContainer();
+  console.log(`Started container ${container.id}`);
```
Comment (mdlavin) [inline test/docker/executeContainerCommand.test.js:7]: Did you mean to make these debug statements instead of console statements?

File: test/docker/executeContainerCommand.test.js [hunk 4]
```diff
@@ -12,7 +13,11 @@ test('response includes stdout, stderr, and inspectOutput with ExitCode 0', asyn
     t.is(stderr.toString('utf8'), '');
     t.is(inspectOutput.ExitCode, 0);
   } finally {
-    // don't wait for container to stop since it could take a while
```
Comment (mdlavin) [inline test/docker/executeContainerCommand.test.js:15]: Is this not true anymore?

File: test/lambda/graphql.test.js [hunk 5]
```diff
@@ -32,9 +32,12 @@ test.before(async () => {
 test.after.always(async () => {
   delete process.env.COMPOSE_PROJECT_NAME;
   try {
-    container.stop();
+    if (container) {
+      await container.stop();
+      console.log(`Stopped container ${container.id}`);
```
Comment (mdlavin) [inline test/lambda/graphql.test.js:37]: Should be a debug statement?

**Review by mdlavin — COMMENTED** [review]
It looks like test/lambda/types.test.ts is an empty file that accidentally got committed

File: test/docker/executeContainerCommand.test.js (hunk 3 above)
Comment (DavidTanner) [reply test/docker/executeContainerCommand.test.js:7]: This is just for tests, not part of the lib.

File: test/docker/executeContainerCommand.test.js (hunk 4 above)
Comment (DavidTanner) [reply test/docker/executeContainerCommand.test.js:15]: I haven't seen it take long, but I have seen that the process can exit before the command is sent to docker, and so the container never stops.

File: test/lambda/graphql.test.js (hunk 5 above)
Comment (DavidTanner) [reply test/lambda/graphql.test.js:37]: This is also a test file, so we aren't using the debug library.

File: src/localstack.ts (hunk 2 above)
Comment (DavidTanner) [reply src/localstack.ts:423]: I changed this while hunting down containers that didn't actually get stopped after all the tests finished.

File: src/docker.js (hunk 1 above)
Comment (DavidTanner) [reply src/docker.js:117]: Dang it.
